lib/chef/provider/subversion.rb

```diff
@@ -194,6 +196,21 @@ def authentication
         result
       end
 
+      def proxy
+        %w{http https}.each { |p|
+          if @new_resource.repository.start_with?("#{p}:") &&
+              ENV["#{p}_proxy"] != nil &&
```

Comment:
Huh, I had it in my head it was bi-directional and Config would return ENV values, but it looks not.